Cargo.toml

```diff
@@ -7,32 +7,30 @@ members = [
 ]
 
 [workspace.package]
-rust-version = "1.74.0"
+rust-version = "1.81.0"
```

Comment:
The Core team is still using 1.74 because of some WASM incompatibilities I guess (e.g. [thread](https://stellarfoundation.slack.com/archives/C030Z9EHVQE/p1727811768128959?thread_ts=1727760416.953479&cid=C030Z9EHVQE)), so we should keep the same one here too to be extra safe.

Reply:
env is pointing at 1.79: https://github.com/stellar/rs-soroban-env/blob/cf469d2d4be82f83c24b75aeb8a133cfb82a23cd/Cargo.toml#L24